From cdc4bc6fa37e3d5173b3cad3d0f2e2d9816f8eb1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Sun, 15 Dec 2024 16:13:42 +0100
Subject: [PATCH 1/2] Drop versioned dependencies

Upstream bumps version requirements arbitrarily, in most cases without
any specific API or functional requirement.
Use package version available in the distribution instead.
---
 requirements/base.txt   | 12 ++++++------
 requirements/zeromq.txt |  3 +--
 2 files changed, 7 insertions(+), 8 deletions(-)

diff --git a/requirements/base.txt b/requirements/base.txt
index 0061087e35..a42ff50f7d 100644
--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -10,21 +10,21 @@ distro>=1.0.1
 psutil>=5.0.0
 packaging>=21.3
 looseversion
-tornado>=6.3.3
-aiohttp>=3.9.0
+tornado
+aiohttp
 
 # We need contextvars for salt-ssh.
 # Even on python versions which ships with contextvars in the standard library!
 contextvars
 
 setproctitle>=1.2.3
-timelib>=0.2.5
-pyopenssl>=24.0.0
+timelib
+pyopenssl
 python-dateutil>=2.8.1
 python-gnupg>=0.4.7
 cherrypy>=18.6.1
 importlib-metadata>=3.3.0
-cryptography>=42.0.0
+cryptography
 
 # From old requirements/static/pkg/linux.in
 rpm-vercmp; sys_platform == 'linux'
diff --git a/requirements/zeromq.txt b/requirements/zeromq.txt
index 4000f5eb01..ec03558d33 100644
--- a/requirements/zeromq.txt
+++ b/requirements/zeromq.txt
@@ -1,4 +1,4 @@
 -r base.txt
 -r crypto.txt
 
-pyzmq>=25.1.2
+pyzmq
-- 
2.46.0

